app/uk/gov/hmrc/slacknotifications/services/UserManagementService.scala

```diff
@@ -34,6 +34,15 @@ class UserManagementService @Inject()(
   ec: ExecutionContext
 ) extends Logging {
 
+  def getTeamsForLdapUser(ldapUsername: String)(implicit hc: HeaderCarrier):Future[List[TeamDetails]] =
+    for {
+      teams     <- connector.getTeamsForUser(ldapUsername)
+      teamsFound = teams match {
+                    case Nil => logger.warn(s"No teams found for ldap username: '$ldapUsername'")
+                    case _   => logger.info(s"Teams found for ldap username: '$ldapUsername' are ${teams.mkString("[", ",", "]")}")
```

Comment:
I'd put this at debug level

Reply:
Can `getTeamsForGithubUser` now call `getLdapUsername` and `getTeamsForLdapUser`?
Should the logging go in `NotificationService` which has the context of whether it's a concern or not?